problem_builder/management/commands/copy_deprecated_course_id.py

```diff
@@ -0,0 +1,45 @@
+import time
+from django.core.management.base import BaseCommand
+from problem_builder.models import Answer
+
+
+class Command(BaseCommand):
+    """
+    Copy content of the deprecated Answer.course_id column into Answer.course_key in batches.
+    The batch size and sleep time between batches are configurable.
+    """
+    help = 'Copy content of the deprecated Answer.course_id column into Answer.course_key in batches'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch-size',
+            help='The size of each batch of records to copy (default: 5000).',
+            type=int,
+            default=5000,
+        )
+        parser.add_argument(
+            '--sleep',
+            help='Number of seconds to sleep before processing the next batch (default: 5).',
+            type=int,
+            default=5
+        )
+
+    def handle(self, *args, **options):
+        batch_size = options['batch_size']
+        sleep_time = options['sleep']
+        queryset = Answer.objects.filter(course_key__isnull=True)
+        batch_count = (queryset.count() / batch_size) + 1
```

Comment:
Nit: This should do a floating point division with a ceiling operation instead - otherwise cases where the queryset length is divisible by the batch_size will result in one additional batch count than necessary. E.g.:

```python
batch_count = int(math.ceil(queryset.count() / float(batch_size)))
```